operator/api/v1alpha1/manifest_types.go

```diff
@@ -42,6 +42,11 @@ type InstallInfo struct {
 
 // ManifestSpec defines the specification of Manifest.
 type ManifestSpec struct {
+	// Remote indicates if Manifest should be installed on a remote cluster
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=true
```

Comment:
This will cause issues if you try to set it to false since it also defaults it back to true. If thats on purpose all good but we had problems with this in lifecycle manager